tests/integration/go_ethereum/common.py

```diff
@@ -10,45 +10,47 @@
     Web3ModuleTest,
 )
 
+OLDER_GETH_VERSIONS = ['v1.7.2', 'v1.8.22']
+NEWER_GETH_VERSIONS = ['v1.8.22', 'v1.9.7']
```

Comment:
Having `v1.8.22` in both of these is a little awkward.  I see why once I look through the code below but maybe this would be easier to grok if it was done as:

```python
VERSIONS_SUPPORTING_XXX = ...
VERSIONS_WITHOUT_YYY = ...
```